server/game/cards/characters/03/kingrobbshost.js

```diff
@@ -0,0 +1,53 @@
+const _ = require('underscore');
+
+const DrawCard = require('../../../drawcard.js');
+
+class KingRobbsHost extends DrawCard {
+    setupCardAbilities() {
+        this.reaction({
+            when: {
+                afterChallenge: (event, challenge) => (
+                    challenge.challengeType === 'military' &&
+                    challenge.winner === this.controller &&
+                    challenge.isParticipating(this) &&
+                    challenge.loser.faction.power >= 1)
+            },
+            handler: () => {
+                this.game.promptForSelect(this.controller, {
+                    activePromptTitle: 'Choose an attacking character',
+                    source: this,
+                    cardCondition: card => card.location === 'play area' && card.getType() === 'character' && this.game.currentChallenge.isAttacking(card),
+                    onSelect: (p, card) => this.onCardSelected(p, card)
+                });
+            }
+        });
+    }
+
+    onCardSelected(p, card) {
+        var loser = this.game.currentChallenge.loser;
+
+        if(loser.faction.power >= 2) {
+            var power = this.anyPlotHasTrait('War') ? 2 : 1;
+        } else {
+            power = 1;
+        }
+
+        this.game.addPower(loser, -power);
+        card.modifyPower(power);
+
+        this.game.addMessage('{0} uses {1} to move {2} power from {3}\'s faction card to {4}', 
+                              this.controller, this, power, loser, card);
+                        
+        return true;
+    }
+
+    anyPlotHasTrait(trait) {
```

Comment:
Feels like this should be a method on game, in case anything else wants to do a similar check (of which I'm sure there are a few)